### PR TITLE
logback v0.1.2

### PR DESCRIPTION
**Pull request by sumitjoshi1989**
Gradle changes to support latest gradle.
Code changes to support latest java and javabrake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,19 +20,20 @@ Maven:
...
 
 ```xml
-<dependency org='io.airbrake' name='logback' rev='0.1.1'>
+<dependency org='io.airbrake' name='logback' rev='0.1.2'>
   <artifact name='logback' ext='pom'></artifact>
 </dependency>
 ```
 
 ## Configuration
+If you want to send the error logs to Airbrake, you need to have following lines in logback.xml. Add this file in resources folder. This file is main file for logback configuration. and contains information about log levels, log appenders.
```
Comment (mmcdaris) [inline README.md]: Small improvements
```suggestion
If you want to send the error logs to Airbrake, you need to have following lines in logback.xml. Add this file in the resources folder. This is the main file for logback configuration and contains information about log levels, log appenders.
```

File: README.md [hunk 2]
```diff
@@ -53,3 +58,48 @@ Ivy:
   </root>
 </configuration>
 ```
+## Error Logging
+
+```java
+import org.slf4j.LoggerFactory;
+import ch.qos.logback.classic.Logger;
+
+ Logger logger = ((ch.qos.logback.classic.Logger))LoggerFactory.getLogger("Name");
```
Comment (mmcdaris) [inline README.md]: Tiny whitespace edit:
```suggestion
Logger logger = ((ch.qos.logback.classic.Logger))LoggerFactory.getLogger("Name");
```

File: src/test/java/io/airbrake/logback/MockSyncSender.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    
+  }
+
+  @Override
+  public void setAPMHost(String host) {
+    // TODO Auto-generated method stub
+    
+  }
+
+
```
Comment (mmcdaris) [inline src/test/java/io/airbrake/logback/MockSyncSender.java]: Remove extra newlines
```suggestion
```

**Review by mmcdaris — APPROVED** [review]
Nice update @sumitjoshi1989, the README is looking a lot better with your changes as well.

**Review by abhishekpargal12 — APPROVED** [review]
looks good

**Review by mmcdaris — APPROVED** [review]
Nice one @sumitjoshi1989, Some comments I left earlier could still be addressed but not a blocker.

**Pull request merged**